src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -817,6 +857,10 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             _touchAnchor = std::nullopt;
         }
 
+        _autoScrollVelocity = 0;

```

Comment:
Do we need to wrap this in a `PointerDeviceType` check to make sure we're using a mouse?

Reply:
Yes, indeed. But even better, let's check if this is the same pointer that initiated the scroll.

Reply:
Just realized this should be the same logic as previously to stop the scroll. I'mma make this a function.